### DISCUSSION TITLE
Remove an existing conda environment

### DISCUSSION THREAD

**Discussion by filippo82** [nebari-dev/nebari#528] [Q&A] [answered]
Removing an environment from `qhub-config.yaml` does not remove it from the conda-store container. Using `k9s`, I can log in into the container conda-store container and the removed environment is still under `/home/conda/store`.

Is this expected?

Thanks!

**Comment by tylerpotts** [comment, marked as answer]
I'd say this is expected, but not ideal. Eventually we will be transitioning to conda store which will fix things like this: https://github.com/Quansight/conda-store

**Reply by filippo82** [reply]
Isn't the conda store already used? https://github.com/Quansight/qhub-terraform-modules/blob/main/modules/kubernetes/services/conda-store/main.tf

**Reply by tylerpotts** [reply]
...very late reply. Sorry for the delay

There is a pod in kubernetes called conda store, but it is the alpha proof-of-concept level implementation that uses a network file system to allow all pods access to the conda environments. 

@costrouc Wrote a more advanced implementation that leverages the structure of Docker and lessons learned from Nix-OS (which is the code [here](https://github.com/Quansight/conda-store)). We working on finding the right time and resources to implement this into QHub.

**Comment by Adam-D-Lewis** [comment]
I'm closing old discussions in a repo cleanup effort.  Feel free to re-open if needed.
